### PR TITLE
Feature/jshinthook

### PR DESCRIPTION
**Pull request by kejace**
Just added it, I haven't actually cleaned up any code so that jshint will pass.

Note that you can bypass a check passing the `-n` flag to `git`, ie `git commit -am "commit message" -n`.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "asi"           : true,
+  "laxbreak"      : true,
+  "bitwise"       : true,
+  "boss"          : false,
+  "curly"         : true,
+  "eqeqeq"        : true,
```
Comment (liorsaar1) [inline .jshintrc]: test comment

File: .jshintrc [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "eqnull"        : false,
+  "evil"          : false,
+  "expr"          : false,
+  "forin"         : false,
+  "immed"         : true,
+  "indent"        : 2,
+  "latedef"       : true,
+  "loopfunc"      : false,
+  "noarg"         : true,
```
Comment (liorsaar1) [inline .jshintrc]: another test comment ?

Comment (kejace) [inline .jshintrc]: this works (from email)

**Comment by liorsaar1** [comment]
Looks good.

**Pull request merged**